pkg/api/types.go

```diff
@@ -55,7 +55,7 @@ import (
 //      Hypens ('-') cannot be leading or trailing character of the string
 //      and cannot be adjacent to other hyphens.
 
-// ObjectMeta is metadata that all persisted resources must have, which includes all objects
+// Represents metadata that all persisted resources must have, which includes all objects
```

Comment:
I would rewrite this one as `A reference to an object or piece of an object.`, because what's here now doesn't sound like the right tone for this doc.